collections/copernicus-dem.yaml

```diff
@@ -138,9 +138,9 @@ Summaries:
       data_type: float32
       statistics:
         minimum:
-          - 0
+          - -413
```

Comment:
Where is this data from? I found multiple sources indicating that the value is even lower (420 / 430m below sea level).